MapboxDirections/MBAttribute.h

```diff
@@ -1,17 +1,41 @@
 /**
- Attributes are metadata information for a given route. The number of attributes returned will be a direct 1-1 relationship with the route's full geometry. The `.distance`, `.expectedTravelTime`, and `.speed` attributes have one fewer value than the `.openStreetMapNodeIdentifier` attribute.
-*/
+ Attributes are metadata information for a route leg.
+ 
+ When most of the attributes are specified, the resulting route leg contains one attribute value for each segment in leg, where a segment is the straight line between two coordinates in the route leg’s full geometry. When the `MBAttributeOpenStreetMapNodeIdentifier` attribute is specified, the corresponding `RouteLeg` property contains one more value than each of the other attributes.
+ */
 typedef NS_OPTIONS(NSUInteger, MBAttributeOptions) {
-
-    /// Segment distance. Distances are measured in meters.
-    MBAttributeDistance = (1 << 1),
+    /**
+     [OpenStreetMap node identifier](https://wiki.openstreetmap.org/wiki/Node).
+     
+     When this attribute is specified, the `RouteLeg.openStreetMapNodeIdentifiers` property contains one value for each coordinate in the leg’s full geometry.
+     */
+    MBAttributeOpenStreetMapNodeIdentifier = (1 << 1),
```

Comment:
I’ve changed the underlying values of the various attributes for forwards-compatibility. Unfortunately, this is sort of a breaking change in that developers who rely on the raw values of these attribute options may end up with attributes they don’t expect.